mbof/models.py

```diff
@@ -21,7 +21,6 @@ def currentUserObject():
     return User.objects.get(loginName=os.getenv('REMOTE_USER'))
 
 
-@python_2_unicode_compatible
```

Comment:
Ah, good call. We haven't decided what we are doing with Role yet, but I will update it to unicode in the meantime. Thanks!